xml/System.Security.Cryptography/DSAOpenSsl.xml

```diff
@@ -80,9 +113,22 @@
         <Parameter Name="handle" Type="System.IntPtr" />
       </Parameters>
       <Docs>
-        <param name="handle">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="handle">The OpenSSL <c>DSA*</c> value to use as the key.</param>
```

Comment:
What kinds of values is `<c>DSA*</c>` referring to? Would users benefit on having a list in the remarks with the possible values this can acquire?

Reply:
A pointer to a DSA value.  `DSA*`.

Reply:
Oh it's a pointer, I thought it was a wildcard. Got it.